ftest-toxic-frontend-reporter/src/test/java/org/arquillian/cube/q/toxic/reporter/ToxicFunctionalTest.java

```diff
@@ -0,0 +1,76 @@
+package org.arquillian.cube.q.toxic.reporter;
+
+import org.arquillian.cube.HostIp;
+import org.arquillian.cube.impl.util.IOUtil;
+import org.arquillian.cube.q.api.NetworkChaos;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+
+import java.io.IOException;
+import java.net.SocketException;
+import java.net.URL;
+
+import static org.arquillian.cube.q.api.NetworkChaos.DistributedLatencyType.logNormalLatencyInMillis;
+import static org.arquillian.cube.q.api.NetworkChaos.RateType.rate;
+import static org.arquillian.cube.q.api.NetworkChaos.TimeoutType.timeoutInMillis;
+import static org.arquillian.cube.q.api.Q.IterationRunCondition.times;
+
+@RunWith(Arquillian.class)
+public class ToxicFunctionalTest {
```

Comment:
Shouldn't this class be a test of functionality implemented in the reporter(recorder) module? 
This whole module seems to be very similar to this one: https://github.com/arquillian/arquillian-cube-q/tree/master/ftest-toxic-frontend so it could be considered as a duplication. I know that the dependencies are different (in this module are added the reporter deps), but the test is targeted on Cube Q logic, not on the reporter features.
In addition, I'm asking myself why we need a separated module for reporter ftest?